source/level_set_okz.cc

```diff
@@ -462,8 +461,17 @@ template <int dim>
 void
 LevelSetOKZSolver<dim>::compute_curvature(const bool diffuse_large_values)
 {
-  TimerOutput::Scope timer(*this->timer, "LS compute curvature.");
-  curvatur_operator->compute_curvature(diffuse_large_values);
+  // This function computes the curvature from the normal field. Could also
+  // compute the curvature directly from C, but that is less accurate. TODO:
+  // include that variant by a parameter
+  {
+    TimerOutput::Scope timer(*this->timer, "LS compute normal.");
+    this->compute_normal(false);
+  }
+  {
+    TimerOutput::Scope timer(*this->timer, "LS compute curvature.");
+    curvatur_operator->compute_curvature(diffuse_large_values);
+  }
```

Comment:
... instead we need to call the normal operator ourselves.

Reply:
I will follow the same strategy in meltpool.